### DISCUSSION TITLE
2022-09-21 Weekly Meeting 74 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#296] [Weekly Meetings]
Weekly Meeting 74 is happening on Wednesday September 21st, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research: TBD](https://github.com/omigroup/ux-research) (setting up the repo) 
  - UX vote #222 is approved and the repo is formed, do we have the first meeting? 
- any others?

### Discussions: 
- Retro: What it is, and the many ways to run one.  Live Workshop. 

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
# Retro 

![retroomi-0921](https://user-images.githubusercontent.com/63426722/191554173-017e8bfa-ed2f-4dfa-8901-1affbcb3b50a.png)

Let's have a retro using our tools! We will start with a quick review of what they are, and how to conduct them. This is the best way to generate new priorities and share opportunities for improvement internally and externally.

Retros are a safe space, and the heartbeat of iteration. 

Quick Review: 
- other tools  exist:
    - https://www.reetro.app/
- Using our own tools (github for now) we can create a new board under our omigroup org, and close it out when we finish: https://github.com/orgs/omigroup/projects?query=is%3Aopen 
    - Let's create a board and setup our columns to be something _like_ this: 
      - ![image](https://user-images.githubusercontent.com/63426722/191546321-94ea25c7-aa22-45b8-8297-04abfea1a643.png)
      - :heavy_check_mark: You can find our [retro board here](https://github.com/orgs/omigroup/projects/10/views/1) :point_left: 
    - this lets us openly share ideas as sticky notes (white text and it says "draft")
    - we can create action items out of anything we need to by converting those draft items into issues, and add them to the holistic board. 
    - we can then close the board! It will stay in archive for future reference.

**Comment by mrmetaverse** [comment]
# Developer Hour 
- Scheduled for tomorrow at 17:00 UTC
![image](https://user-images.githubusercontent.com/63426722/191557675-28110f35-2dab-44b1-8d8b-6962fbeb7880.png)

**Reply by mrmetaverse** [reply]
:calendar: https://calendar.google.com/calendar/u/0/embed?src=c_0dpugf9f83eq4qekmb6omqbjmg@group.calendar.google.com

**Comment by mrmetaverse** [comment]
# gLTF extensions 
- meeting tomorrow 
- [tomorrows agenda](https://github.com/omigroup/gltf-extensions/discussions/107)
- [last meetings notes](https://github.com/omigroup/gltf-extensions/discussions/106)

**Comment by mrmetaverse** [comment]
Maybe we have an all-subgroups meeting. Or tags to answer questions. To be found easily.

**Comment by mrmetaverse** [comment]
# Media group 
- next meeting is next month 
- A. offered to stream our content to Youtube, we need to get them access.

**Comment by mrmetaverse** [comment]
# Website 
- pending change for adding youtube link

**Comment by mrmetaverse** [comment]
# MSF Delegates 
[09/21/22 Metaverse Standards Forum: Oversight #29](https://github.com/omigroup/MSF-Delegates/discussions/29)
- The creation for the exploratory group was approved 
![image](https://user-images.githubusercontent.com/63426722/191559833-1306719e-b27d-46dd-91b4-c80fd20b2bbf.png)

**Comment by mrmetaverse** [comment]
# UX Research 
- ![image](https://user-images.githubusercontent.com/63426722/191560378-567eeda7-0fbb-4a48-a57b-7373ce05daa9.png)
- team is looking for a kickoff time
- still looking for volunteers to participate and co-organize
